src/helpers/dataframe/types.ts

```diff
@@ -36,6 +36,9 @@ export interface DataFrame<M extends Obj = Obj, C extends Obj = Obj> {
   columnDescriptors: readonly ColumnDescriptor<C>[]
   metadata?: M
 
+  // If true, only one column can be sorted at a time, and any update to orderBy will replace the previous one.
```

Comment:
Add the default value in the comment (`false`). It's fairly obvious, but I think it's better if it's explicit